chore(jans-core): notify-client is not included in parent pom.xml

**Describe the bug**

notify-client is not included in parent pom.xml of jans-core. Which leads to issues when flip version.

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:4.0.0-M7:site (default-site) on project jans-auth-server-parent: Failed to get report for org.apache.maven.plugins:maven-javadoc-plugin: Failed to execute goal on project jans-auth-server: Could not resolve dependencies for project io.jans:jans-auth-server:war:1.0.15: Could not find artifact io.jans:jans-notify-client:jar:1.0.15 in otp_auth-local (file:///home/jetty/.jenkins/workspace/jans-auth-server/jans-auth-server/server/integrations/otp/repository) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

**To Reproduce**
Flip version and trigger full rebuild.

--- a/jans-core/pom.xml
+++ b/jans-core/pom.xml
@@ -62,6 +62,7 @@
 		<module>demo-cdi</module>
 		<module>doc</module>
 		<module>uma-rs-core</module>
+		<module>notify-client</module>
 	</modules>
 
 	<dependencyManagement>

